src/SDKs/Media/Management.Media/Generated/Models/StreamingLocatorUserDefinedContentKey.cs

```diff
@@ -1,80 +0,0 @@
-// <auto-generated>
```

Comment:
@quintinb Apologize for missing this. Deleting a model type could be a breaking change to users. Please bump the version to `2.0.0` in the csproj
I also see that the API version is preview but the SDKs generated are stable, this is usually not recommended.
@shahabhijeet Please confirm

Reply:
I bumped the version number as requested.  Generating the SDKs as stable was not intentional.  Can you advise on how to generate them as preview next time?  Is it by adding preview to the version string instead of just numbers?  Thanks.